src/vdaf/prio3.rs

```diff
@@ -1335,7 +1464,7 @@ where
 
         Ok((
             Prio3PrepareState {
-                measurement_share: msg.measurement_share.clone(),
+                measurement_share: msg.measurement_share().clone(),
```

Comment:
This method already clones the vector/seed, so we can remove a clone here.
```suggestion
                measurement_share: msg.measurement_share(),
```